### ISSUE TITLE
Improve contrast ratio for comments

### ISSUE THREAD

**Issue by mattrobineau** [hardhackerlabs/themes#3]
Assuming the goal for this theme is to have good readability, I would suggest changing the comment's colour or intensity.

The current contrast between comments and the background is considered a fail according to [WebAIM](https://webaim.org/resources/contrastchecker/?fcolor=777383&bcolor=282433).

**Comment by anson0370** [comment]
Thanks a lot for your suggestion. I forgot to check the contrast ratio of the comments. Now it meets the WCAG AA standard (although it still doesn't meet the WCAG AAA standard - I feel it's too bright and difficult to distinguish from regular text). We're updating it in all the themes.

After we complete all the adjustments, I will close this issue.
